ethereum_history_api/oracles/package.json

```diff
@@ -14,6 +14,7 @@
     "format": "prettier --write src",
     "format:ci": "prettier --check src",
     "oracle-server": "tsx src/script/startOracleServer",
+    "prepare-js-fixtures": "tsx src/script/prepareJSFixtures",
```

Comment:
Script naming inconsistency `prepare...` vs `fetch...`

Reply:
We'll stop using fetch in the next PR. I'll leave it for now